docs/docs/release-notes.md

```diff
@@ -2,8 +2,10 @@
 
 ## 1.4
 
-1. EPID attestation protocol has been deprecated, and it will be removed in an upcoming release. You should use DCAP instead.
-2. For security reasons, the way [enclave constraints](api/-conclave%20-core/com.r3.conclave.common/-enclave-constraint/index.html)
+1. Intel SGX SDK has been updated to 2.18. This provides bug fixes, security updates, and other improvements. See the
+    [SGX SDK release notes](https://github.com/intel/linux-sgx/releases) for more details.
```

Comment:
Use the specific linke for 2.18: https://github.com/intel/linux-sgx/releases/tag/sgx_2.18